test/testFill.cpp

```diff
@@ -151,24 +151,31 @@ TEST_CASE("Radial Filling", "[tvgFill]")
     auto fill = RadialGradient::gen();
     REQUIRE(fill);
 
-    float cx, cy, radius;
+    float cx, cy, r, fx, fy, fr;
 
-    REQUIRE(fill->radial(0, 0, -1) == Result::InvalidArguments);
-    REQUIRE(fill->radial(nullptr, nullptr, nullptr) == Result::Success);
-    REQUIRE(fill->radial(100, 120, 50) == Result::Success);
+    REQUIRE(fill->radial(0, 0, -1, 0, 0, 0) == Result::InvalidArguments);
+    REQUIRE(fill->radial(0, 0, 0, 0, 0, -1) == Result::InvalidArguments);
+    REQUIRE(fill->radial(nullptr, nullptr, nullptr, nullptr, nullptr, nullptr) == Result::Success);
+    REQUIRE(fill->radial(100, 120, 50, 10, 20, 5) == Result::Success);
 
-    REQUIRE(fill->radial(&cx, nullptr, &radius) == Result::Success);
+    REQUIRE(fill->radial(&cx, nullptr, &r, nullptr, nullptr, nullptr) == Result::Success);
```

Comment:
let's remove the nullptr passes for focal.

Reply:
please check now, removed also in l.158